src/locale/languages/es.js

```diff
@@ -221,6 +221,9 @@ export const erase = {
       deleted: 'DELETED',
       fileRenamed: 'File renamed without any changes',
     },
+    readMe: {
+      noReadMeFound: 'No README.md found',
```

Comment:
Could you add the `readMeActions` key? It exist in en.js :)

Reply:
Yeah, the hole `readMe` part was missing in es, not sure why.